src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/cargo/CargoUtils.java

```diff
@@ -69,21 +69,7 @@ static boolean hasInventory(@Nullable Block block) {
         }
 
         Material type = block.getType();
-
-        // TODO: Add designated SlimefunTag
-        return switch (type) {
-            case CHEST,
-                TRAPPED_CHEST,
-                FURNACE,
-                DISPENSER,
-                DROPPER,
-                HOPPER,
-                BREWING_STAND,
-                BARREL,
-                BLAST_FURNACE,
-                SMOKER -> true;
-            default -> SlimefunTag.SHULKER_BOXES.isTagged(type);
-        };
+        return SlimefunTag.INVENTORY_BLOCKS.isTagged(type);
```

Comment:
This tag does not necessarily represent all storage blocks. Only the ones cargo supports. The tag name should reflect that.

Reply:
I thought let me generalize them so it can be used on multiple places.
At first it was like you wanted XD